index.js

```diff
@@ -136,36 +200,47 @@ function nowServing(line) {
 }
 
 const line = []
+let ticketNumber = 0 
+function takeATicketNumber(line){
+  //let ticketNumber = 0  will start 0 every time
+   ticketNumber ++
+   line.push(ticketNumber)
+   return `Welcome. You are ticket number ${ticketNumber}.`
```

Comment:
Logical semantic naming, good use of camel case for variable and function names. Correct logic 💯